components/core/src/clp/ffi/SchemaTree.hpp

```diff
@@ -167,7 +167,9 @@ class SchemaTree {
      * Inserts a new node corresponding to the given locator.
      * @param locator
      * @return The ID of the inserted node.
-     * @throw OperationFailed if a node that corresponds to the given locator already exists.
+     * @throw OperationFailed if either:
```

Comment:
```suggestion
     * @throw OperationFailed if:
```
As discussed offline